models/schedule_item.go

```diff
@@ -119,7 +119,7 @@ func getBlock(name string, StartTime time.Time) string {
 
 		{"roses live 20", "event"},
 		{"roses 20", "event"},
-    {"freshers 20", "event"},
+    	{"freshers 20", "event"},
```

Comment:
Erroneous whitespace, double check that you have gofmt setup as it should have caught this for you.